src/bionics.h

```diff
@@ -156,6 +158,30 @@ struct bionic {
     void deserialize( JsonIn &jsin );
 };
 
+class BionicsDisplayType
+{
+        friend class string_id<BionicsDisplayType>;
+        bionics_displayType_id _ident;
+        translation _display_string;
+    public:
+        static std::vector<BionicsDisplayType> displayTypes;
+        static void load( JsonObject &jsobj );
+        static void reset();
+        static bionics_displayType_id infer_type( const bionic_data &b );
+
+        static const BionicsDisplayType &get_display_type( bionics_displayType_id );
```

Comment:
```suggestion
        static const BionicsDisplayType &get_display_type( const bionics_displayType_id &id );
```
No need to copy the id. And the clang-tidy thingy will (probably) complain when the name of parameter in the declaration does not match the name in the definition.

Reply:
Turns out this function is not used at all, so I just deleted it =P